src/web/lib/decidePalette.ts

```diff
@@ -13,6 +14,7 @@ const WHITE = neutral[100];
 const BLACK = text.primary;
 
 const textHeadline = (format: Format): string => {
+	if (format.theme === Special.SpecialReport) return specialReport[300];
```

Comment:
This could grow into a nested switch to make it more scalable bot for now this simple exception seems logical